scripts/screeps_rl_experiment_card.py

```
@@ -145,15 +236,243 @@ def build_card(
+
+
+def construction_priority_registry_defaults(registry_path: Path) -> dict[str, JsonObject]:
+    defaults = {
+        variant_id: dict(parameters)
+        for variant_id, parameters in CONSTRUCTION_PRIORITY_FALLBACK_DEFAULTS.items()
+    }
+    try:
+        import screeps_rl_training_runner as training_runner
+
+        registry = training_runner.load_strategy_registry(registry_path)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)